<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectingDialog</class>
 <widget class="QDialog" name="ConnectingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="topMargin">
    <number>30</number>
   </property>
   <item>
    <widget class="QPushButton" name="connectAnyButton">
     <property name="text">
      <string>Connect to any</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="connectAddressButton">
       <property name="text">
        <string>Connect by address</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="addressEdit">
       <property name="text">
        <string>FE:A3:CB:06:AC:01</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
